quickwit/quickwit-opentelemetry/src/otlp/logs.rs

```diff
@@ -60,13 +60,17 @@ doc_mapping:
     - name: timestamp_nanos
       type: datetime
       input_formats: [unix_timestamp]
+      # should be this but not supported
```

Comment:
Not relevant. `unix_timestamp` means quickwit detect automatically if it's in seconds or something else.